src/shared/util.js

```diff
@@ -40,6 +40,13 @@ export const parseUTCDate = (dateStr: string) => {
   return moment.utc(dateStr);
 };
 
+export const parseLocalDate = (dateStr: string) => {
+  return moment(new Date(dateStr));
+};
+export const localToISOString = (d: Date)=> {
+  return moment(d).format('YYYY-MM-DDT00:00:00.000Z');
```

Comment:
From the moment documentation the given format is the same as the default format. Not sure if we should remove the format, refer to `moment.defaultFormat` or keep it as is. What do you think?
https://momentjs.com/docs/#/displaying/format/